source/jsonnet/england-wales/individual/blocks/employment/main_job_introduction.jsonnet

```
@@ -11,7 +11,13 @@ local rules = import 'rules.libsonnet';
         title: 'Main job',
         contents: [
           {
-            description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours.',
+            description: 'The next set of questions is about your main job. Your main job is the job in which you work the most hours.',
+          },
+          {
+            description: '<strong>Coronavirus (COVID-19)</strong>',
+          },
+          {
+            description: 'Your circumstances may have changed during the coronavirus pandemic. Answer all questions based on your situation as it is now.',
```

This first description should be a heading, and the title and heading should be in the same content block:
```suggestion
          {
            title: 'Coronavirus (COVID-19)',
            description: 'Your circumstances may have changed during the coronavirus pandemic. Answer all questions based on your situation as it is now.',
```